auctions/models.py

```diff
@@ -479,7 +481,9 @@ def fees(self):
         return PayoutFee.objects.filter(payout=self)
 
     def request(self):
-
+        receiver = (
+            paypay_recipient if paypay_recipient else self.claim.user.email
```

Comment:
`paypal_` ... you're subconsciously typing "pay" so many times - I can tell you're very excited about the payments.